discovery-provider/plugins/pedalboard/apps/relay/src/types/relay.ts

```diff
@@ -0,0 +1,23 @@
+import { Static, Type } from "@sinclair/typebox";
+
+export const RelayRequest = Type.Object({
+  senderAddress: Type.String(),
+  encodedABI: Type.String(),
+  gasLimit: Type.Integer(),
+});
```

Comment:
Identity relay had two other params, contract address and registry name which I figured could actually be passed through config since this relay only deals in entity manager. This way older SDKs could still pass those params but they'll be ignored here.